### ISSUE TITLE
Add timestamp for favourites

### ISSUE THREAD

**Issue by ashyon** [tootsuite/mastodon#1608]
Add timestamp for when a toot was favourited in the top right corner like on statuses

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by FiXato** [comment]
I agree this would be useful data which even seems to be missing from the favourites API endpoint.
The same goes for bookmarks.

**Comment by mjankowski** [comment]
Is the that the timestamp of when favourited would *replace* the status creation timestamp? ... seems confusing...

If not, can you add a screenshot showing where in current interface you'd want this information and that its still missing in current/4.3 version.
